u-mad-bro/src/App.js

```diff
@@ -10,10 +10,18 @@ import PrivateRoute from "./components/PrivateRoute.js";
 
 function App() {
   return (
+<<<<<<< HEAD
+    <div className="App">
+      <header className="App-header">
+        
+      </header>
+    </div>
+=======
```

Comment:
Doesn't the stuff below belong INSIDE the "App" div?

Reply:
This was a mistake leftover from the boilerplate I deleted. I handled the merge conflict and this should be taken care of now.